### PR TITLE
Project creation locks in cluster environment #8100

### PR DESCRIPTION
**Pull request by vbradnitski**

### CODE REVIEW

File: modules/core/core-api/src/main/java/com/enonic/xp/init/Initializer.java [hunk 1]
```diff
@@ -17,19 +17,22 @@
...
             builder.initializationCheckMaxCount == null ? INITIALIZATION_CHECK_MAX_COUNT : builder.initializationCheckMaxCount;
+        this.forceInitialization = builder.forceInitialization;
     }
 
     public void initialize()
     {
         while ( true )
         {
-            if ( isMaster() )
+            if ( isMaster() || this.forceInitialization )
```
Comment (rymsha) [inline modules/core/core-api/src/main/java/com/enonic/xp/init/Initializer.java:35]: switch them. checking boolean is faster than checking isMaster

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/core/core-api/src/main/java/com/enonic/xp/init/Initializer.java  1
         

Clones removed
==============
+ modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5551523&bid=18202096)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/core/core-api/src/main/java/com/enonic/xp/init/Initializer.java  1
         

Clones removed
==============
+ modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5551523&bid=18202096)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- modules/core/core-api/src/main/java/com/enonic/xp/init/Initializer.java  1
         

Clones removed
==============
+ modules/core/core-api/src/main/java/com/enonic/xp/project/CreateProjectParams.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5551523&bid=18202096)

**Pull request merged**